core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala

```diff
@@ -715,7 +714,7 @@ class FunctionPullingContainerProxy(
       logging.info(
         this,
         s"This is the remaining container for ${data.action}. The container will stop after $warmedContainerKeepingTimeout.")
-      setTimer(KeepingTimeoutName, Remove, warmedContainerKeepingTimeout)
+      startSingleTimer(KeepingTimeoutName, Remove, warmedContainerKeepingTimeout)
```

Comment:
`startTimerAtFixedRate` instead ?

Reply:
I checked the source code
```
  def startTimerAtFixedRate(name: String, msg: Any, interval: FiniteDuration): Unit =
    startTimer(name, msg, interval, FixedRateMode)

  ...
  /** INTERNAL API */
  @InternalApi
  private[akka] case object FixedRateMode extends TimerMode {
    override def repeat: Boolean = true
  }
```
the method: `startTimerAtFixedRate` will invoke startTimer with `repeat: true`,
but for here, it is a one time timer,
So startSingleTimer is ok here.